rll/shutdown-reason.sh

```diff
@@ -31,9 +31,12 @@
 #   reboot
 #   service_restart
 
+# Determine if /usr/local/bin is read-only
```

Comment:
This section is not determining if /usr/local/bin is read-only.  Perhaps:
`# Determine location of rsc`